shepherd/.pylintrc

```diff
@@ -1,2 +1,2 @@
 [MESSAGES CONTROL]
-disable=missing-docstring, no-self-use, too-many-instance-attributes, wildcard-import, unused-wildcard-import, too-many-arguments, too-few-public-methods
+disable=missing-docstring, no-self-use, too-many-instance-attributes, wildcard-import, unused-wildcard-import, too-many-arguments, too-few-public-methods, too-many-return-statements
```

Comment:
I think we don't need to ignore `too-many-return-statements`, we should just avoid it :smiley: